packages/app/src/systems/Send/components/SendConfirm/SendConfirm.tsx

```diff
@@ -13,6 +14,12 @@ export function SendConfirm({ txRequest }: SendConfirmProps) {
   const amountSent = txRequest.ethAmountSent;
   const isDone = txRequest.status('success') || txRequest.status('failed');
   const navigate = useNavigate();
+
+  const goToHome = () => {
+    store.updateAccounts();
```

Comment:
This should be done inside the txRequest machine right after success or failure step.